README.md

```diff
@@ -1,4 +1,33 @@
 # rcsb-training-resources
-This repository contains resources and materials referenced in RCSB PDB training events ([training-events](training-events)) as well as a collection of example files describing various use cases of RCSB.org services ([example-use-cases](example-use-cases)).
+The RCSB PDB frequently offers ([training-events](training-events)) that explain and demonstrate more advanced or automated approaches to utilizing PDB resources. We believe these resources will provide you with tools to improve your research, teaching and overall productivity as explore macromolecular structures. All materials in this repository are open source and governed by the **Do you want to include a license?** license.
+
```

Comment:
We can categorize this content same way we do software: https://www.rcsb.org/pages/policies#usagePolicies. Not sure if there is a license better suited for distribution of educational materials